sdk/frontdoor/arm-frontdoor/README.md

```diff
@@ -34,7 +35,7 @@ const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];
 
 msRestNodeAuth.interactiveLogin().then((creds) => {
   const client = new FrontDoorManagementClient(creds, subscriptionId);
-  client.frontDoors.list().then((result) => {
+  client.networkExperimentProfiles.list().then((result) => {
```

Comment:
is this expected? seems a big change

Reply:
The last version of this package is still which was moved from node sdk repo a year ago. So definitely going to be a big change.